### PR TITLE
[MIG] delivery_package_default_shipping_weight: Migration to 14.0

### PR DESCRIPTION
**Pull request by dzungtran89**

### CODE REVIEW

File: delivery_package_default_shipping_weight/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+{
+    "name": "Delivery Package Default Shipping Weight",
+    "summary": "Set default package shipping weight according to packaging",
+    "version": "14.0.1.0.0",
+    "development_status": "Beta",
```
Comment (rousseldenis) [inline delivery_package_default_shipping_weight/__manifest__.py]: Shouldn't this changed ?

File: delivery_package_default_shipping_weight/models/product_packaging.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+from odoo.tools import float_compare
+
+
+class ProductPackaging(models.Model):
+    _inherit = "product.packaging"
+
+    package_default_shipping_weight = fields.Float(
```
Comment (rousseldenis) [inline delivery_package_default_shipping_weight/models/product_packaging.py:11]: IMHO, we should use digits here as :

https://github.com/odoo/odoo/blob/14.0/addons/product/models/product.py#L119

File: delivery_package_default_shipping_weight/models/stock_quant_package.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+from odoo import api, models
+
+
+class StockQuantPackage(models.Model):
+    _inherit = "stock.quant.package"
+
+    @api.onchange("packaging_id")
+    def _onchange_packaging_id(self):
+        if self.packaging_id and self.packaging_id.package_default_shipping_weight:
+            self.shipping_weight = self.packaging_id.package_default_shipping_weight
```
Comment (rousseldenis) [inline delivery_package_default_shipping_weight/models/stock_quant_package.py:12]: Maybe add a test for this ?

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review. At least digits should be used.

**Comment by simahawk** [comment]
@dzungtran89 ping

**Comment by dzungtran89** [comment]
oups, somehow we missed this :(, thx for pinging me. We'll update it

File: delivery_package_default_shipping_weight/__manifest__.py (hunk 1 above)
Comment (dzungtran89) [reply delivery_package_default_shipping_weight/__manifest__.py]: Hi, should we go for Production/Stable already or wait a few weeks after it's been merged?

**Comment by dzungtran89** [comment]
investigating on the travis build failed

**Comment by dzungtran89** [comment]
fyi, the test case of this module is passed. Travis build is failed by the test case of module `stock_picking_show_return`, somehow [wizard._onchange_picking_id()](https://github.com/OCA/stock-logistics-workflow/blob/14.0/stock_picking_show_return/tests/test_stock_picking_show_return.py#L64) triggers a move's write. I'll create another PR for that module

File: delivery_package_default_shipping_weight/models/product_packaging.py (hunk 2 above)
Comment (dzungtran89) [reply delivery_package_default_shipping_weight/models/product_packaging.py:11]: The code is updated

File: delivery_package_default_shipping_weight/models/stock_quant_package.py (hunk 3 above)
Comment (dzungtran89) [reply delivery_package_default_shipping_weight/models/stock_quant_package.py:12]: I added the test case for this

**Comment by dzungtran89** [comment]
I made a PR https://github.com/OCA/stock-logistics-workflow/pull/891 for the module `stock_picking_show_return`

**Review by simahawk — APPROVED** [review]

File: delivery_package_default_shipping_weight/__manifest__.py (hunk 1 above)
Comment (simahawk) [reply delivery_package_default_shipping_weight/__manifest__.py]: we are using it since more than 1 year on prod... should be ok to switch to stable

Comment (dzungtran89) [reply delivery_package_default_shipping_weight/__manifest__.py]: good day @simahawk , I pushed the update for `development_status`

**Comment by simahawk** [comment]
@rousseldenis good for you?

**Review by rousseldenis — APPROVED** [review]
Code review

**Pull request merged**